composer.json

```diff
@@ -93,7 +93,8 @@
         "vufind-org/vufind-marc": "1.1.0",
         "webfontkit/open-sans": "^1.0",
         "wikimedia/composer-merge-plugin": "2.1.0",
-        "yajra/laravel-pdo-via-oci8": "3.4.0"
+        "yajra/laravel-pdo-via-oci8": "3.4.0",
+        "browscap/browscap-php": "^7.2"
```

Comment:
Might be worth adding this in alphabetical order for consistency.

Reply:
Fixed.